### DISCUSSION TITLE
What's the status quo of getting around bans?

### DISCUSSION THREAD

**Discussion by hl2jeep** [YimMenu/YimMenu#2226] [Q&A]
Recently i was suspended for 30 days in the banwave that happened, i don't really mind losing my items, what i am annoyed about is the thought of having to register new steam accounts and buying the game again if i do get perma banned, therefor im asking in general what the modding community does to get around this, i highly doubt that every modder has stacks of cash to throw at buying new accounts.

**Comment by pierrelasse** [comment]
I had no bans after the banwave on two accounts and on one account i had like the game played through within 1 hour

**Comment by sch-lda** [comment]
I don't know what will happen if you have a rockstar account that also owns other rockstar games like RDR2 using this method. Maybe you will lose rdr2 permanently?

1. Log in to rockstar support website, send a ticket to rockstar customer service requesting the deletion of your rockstar account, provide the required information and reply to the customer service confirmation email.(You may have to wait a few days)
2. Register a new steam account, buy gta5, write down the activation code and refund immediately.(This activation code can only be used on a steam account that owns Gta5, it cannot be given to a steam account that has not purchased gta5 to activate the game ownership)
3. Run gta with your old steam account, follow the instructions to create a new rockstar account, and enter the activation code you got in step 2

There's something mean about this method, but it seems to work

**Comment by ghost** [comment]
Much easier to do the epic games method. Just get the game on epic games and every time you get banned you can create a r* support ticket and tell them to delete your r* account. You can do this as many times as you want and you don't need to re-buy the game every time, which by the way could get you banned from steam or in trouble for requesting so many refunds.

**Reply by hl2jeep** [reply]
Just a question, does epic games ever have sales so I can buy the game cheaper?

**Reply by LintStalker** [reply]
Yes, they have sales from time to time.  I saw it for around $15 a few weeks ago.  You could also buy it from one of those cheap places like cdkeys.   I've bought a few licenses because I didn't know about the epic thing.
